templates/_helpers.tpl

```diff
@@ -61,3 +61,29 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Traverse all binary assets and create path names for them with data
+*/}}
+{{- define "superset.nestedAssets" -}}
+  {{- $root := .Values.assets }}
+  {{- $fringe := list (tuple (list) $root) }}
+  {{- $max := (len (splitList "\n" ($fringe | toPrettyJson))) }}
```

Comment:
Just a bog-standard graph visit, but since we can't have nice things like "while" loops or "break" we need to predetermine the maximum number of nested K/V we'll visit.  There's no function for this, but we *can* get nested JSON which will have a line-per-key at least.